src/main/resources/org/jenkinsci/plugins/docker/workflow/DockerDSL/help.jelly

```diff
@@ -112,6 +112,13 @@
                 Hexadecimal ID of a running container.
             </p>
         </dd>
+        <dt><code>Container.exec([args, command])</code></dt>
```

Comment:
There is only `command`, no `args`, and it is not optional, so I think you meant to write

    Container.exec(command)